core-bundle/tests/EventListener/PreviewUrlCreateListenerTest.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Tests\EventListener;
+
+use Contao\ArticleModel;
+use Contao\CoreBundle\Event\PreviewUrlCreateEvent;
+use Contao\CoreBundle\EventListener\PreviewUrlCreateListener;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\PageModel;
+use Contao\TestCase\ContaoTestCase;
+use PHPUnit\Framework\MockObject\MockObject;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+class PreviewUrlCreateListenerTest extends ContaoTestCase
+{
+    public function testCreatesThePreviewUrlForPages(): void
+    {
+        $requestStack = new RequestStack();
+        $requestStack->push(new Request());
+
+        $event = new PreviewUrlCreateEvent('page', 42);
+
+        /** @var PageModel&MockObject $pageModel */
+        $pageModel = $this->mockClassWithProperties(PageModel::class);
+
+        $adapters = [
+            PageModel::class => $this->mockConfiguredAdapter(['findByPk' => $pageModel]),
+        ];
+
+        $framework = $this->mockContaoFramework($adapters);
```

Comment:
```suggestion
        $framework = $this->mockContaoFramework($adapters);

```

Reply:
The blank line should add a separation between the "arrange" part and the "set"/"act" part of the test, correct?